index.bs

```diff
@@ -6554,6 +6577,98 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheBypass Command ####  {#command-network-setCacheBypass}
+
+The <dfn export for=commands>network.setCacheBypass</dfn> command bypasses the
+network cache for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheBypass = (
+        method: "network.setCacheBypass",
+        params: network.SetCacheBypassParameters
+      )
+
+      network.SetCacheBypassParameters = {
+        bypass: bool,
```

Comment:
Would it make the logic more clear if we change `bypass: bool` to `mode: "bypass" | "default"`?

Reply:
I believe the `bypass: false` actually means not _"do not bypass cache"_, but rather ''remove custom cache behavior"

Reply:
I like this proposal: I think it would make it easier to extend the behavior for bypassing the cache mode if needed.